components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngine.kt

```diff
@@ -654,22 +654,73 @@ class GeckoEngine(
         }
     }
 
+    private fun isFeatureCategoryActive(category: TrackingCategory) =
+            (settings.trackingProtectionPolicy?.contains(category)
+                    ?: false)
```

Comment:
nit: would prefer this to be on one line: ```settings.trackingProtectionPolicy?.contains(category) ?: false)```

Reply:
![image](https://user-images.githubusercontent.com/773158/96928028-f9dabd00-1485-11eb-969a-ec9f586040dc.png)

Unfortunately, `:detekt` complains about it doesn't fit in one line :(


```
style - 5min debt
        MaxLineLength - [isCategoryActive] at /Users/amejiamarmol/Desktop/contributors/amejia481/android-components/components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngine.kt:657:5

```

Reply:
Putting the ?: false in separate line fix the issue :)
![image](https://user-images.githubusercontent.com/773158/96928388-7cfc1300-1486-11eb-90dd-6d60ec54d2a0.png)